.bingo/Variables.mk

```diff
@@ -22,6 +22,12 @@ $(BINGO): .bingo/bingo.mod
 	@echo "(re)installing $(GOBIN)/bingo-v0.2.3"
 	@cd .bingo && $(GO) build -mod=mod -modfile=bingo.mod -o=$(GOBIN)/bingo-v0.2.3 "github.com/bwplotka/bingo"
 
+CONTROLLER_GEN := $(GOBIN)/controller-gen-v0.3.0
```

Comment:
Why 0.3.0? Can you elaborate a bit?

Reply:
This is the version used in the autogenerated `Makefile` of v0.19.4:

```
# find or download controller-gen
# download controller-gen if necessary
controller-gen:
ifeq (, $(shell which controller-gen))
	@{ \
	set -e ;\
	CONTROLLER_GEN_TMP_DIR=$$(mktemp -d) ;\
	cd $$CONTROLLER_GEN_TMP_DIR ;\
	go mod init tmp ;\
	go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0 ;\
	rm -rf $$CONTROLLER_GEN_TMP_DIR ;\
	}
CONTROLLER_GEN=$(GOBIN)/controller-gen
else
CONTROLLER_GEN=$(shell which controller-gen)
endif
```

Reply:
Perfect, sounds reasonable. I wanted to be only super sure here ;)